### PR TITLE
A lot of var logging for airlocks and airlock related things.

### PR DESCRIPTION
**Pull request by Jc1469**
i hate airlocks

### CODE REVIEW

**Comment by ghost** [comment]
good pr. might port that to FTL actually

File: code/datums/wires/airlock.dm [hunk 1]
```diff
@@ -16,7 +16,7 @@
 	..()
 
 /datum/wires/airlock/interactable(mob/user)
-	var/obj/machinery/door/airlock/A = holder
+	var/obj/machinery/door/airlock/A = holder
```
Comment (AshCorr) [inline code/datums/wires/airlock.dm]: :(

**Review by AshCorr — CHANGES_REQUESTED** [review]

File: code/datums/wires/airlock.dm (hunk 1 above)
Comment (Jc1469) [reply code/datums/wires/airlock.dm]: .

**Review by AshCorr — APPROVED** [review]

**Comment by Mrcreepssss** [comment]
wait uhh, why is AI control disabled done twice?

edit: actually i saw it four times...

**Comment by Jc1469** [comment]
Because there's two seperate functions to disable it per instance.  And 2 instances.

**Pull request merged**